### ISSUE TITLE
How i can insert label element after input ?

### ISSUE THREAD

**Issue by vladhd93** [catamphetamine/react-phone-number-input#86]
Hello, is it possible to insert  label element after input ?
thanks !

**Comment by catamphetamine** [comment]
Clarify yourself

**Comment by catamphetamine** [comment]
and provide a picture

**Comment by FakhruddinAbdi** [comment]
I think he needs to handle input error according to the validation result 
<img width="392" alt="screen shot 2017-10-10 at 09 39 30" src="https://user-images.githubusercontent.com/8319873/31371677-1673020c-ad9f-11e7-985f-29a5f0b9eef7.png">

**Comment by vladhd93** [comment]
yes, true, something like this, but we have only placeholder

**Comment by catamphetamine** [comment]
I see, so you need a label not below the whole component but rather below just the phone number input.
Well, one can always add `margin-left` to the label and also style the input border using CSS.

```js
const is_valid = ...
<div className={('phone-number', { 'phone-number--invalid': !is_valid })}>
  <Phone .../>
  {!is_valid && <div>Not correct</div>}
</div>
```

**Comment by catamphetamine** [comment]
Ok, I'll add the label to the library

**Comment by vladhd93** [comment]
cool, thanks =)) between label after input and div with text below component very big difference:
 you can't use selector input:focus ~ label  for example.

**Comment by catamphetamine** [comment]
I released `react-phone-number-input@0.15.0` and it now takes two properties: `error` (a `String`) and `indicateInvalid` flag (set to `true` if the `error` should be displayed, it won't otherwise).

See the updated demo page ("Validation" section).
http://catamphetamine.github.io/react-phone-number-input/
```js
<Phone
  error={!isValidPhoneNumber(value) && 'Invalid phone number'}
  indicateInvalid
  .../>
```
@vladhd93 Does it work for you?
By the way, I wouldn't restrict the phone numbers being entered – see the explanation in the "Validation" section of the demo.

**Comment by Aidosmf** [comment]
@catamphetamine Hello, how can I insert a floatingLabel from react-responsive-ui?
